src/rockstor/system/services.py

```diff
@@ -69,6 +67,9 @@ def init_service_op(service_name, command, throw=True):
         "nut-server",
         "rockstor-bootstrap",
         "rockstor",
+        "replication",
```

Comment:
```suggestion
        "rockstor-replication",
```

These names look to pertain to the systemd names.